docs/src/main/tut/middleware.md

```diff
@@ -197,19 +197,25 @@ We can create a middleware that registers metrics prefixed with a
 provided prefix like this.
 
 ```tut:silent
-import cats.effect.{Resource, IO}
+import cats.effect.{Clock, IO, Resource}
+import org.http4s.HttpRoutes
+import org.http4s.metrics.prometheus.{Prometheus, PrometheusExportService}
+import org.http4s.server.Router
 import org.http4s.server.middleware.Metrics
-import org.http4s.metrics.prometheus.Prometheus
-import io.prometheus.client.CollectorRegistry
 ```
 ```tut:book
 implicit val clock = Clock.create[IO]
 
-val meteredRoutes: Resource[IO, HttpRoutes[IO]] =
+def meteredRouter(routes: HttpRoutes[IO]): Resource[IO, HttpRoutes[IO]] =
   for {
     registry <- Prometheus.collectorRegistry[IO]
     metrics <- Prometheus.metricsOps[IO](registry, "server")
-  } yield Metrics[IO](metrics)(apiService)
+    metricsSvc <- PrometheusExportService.build[IO]
```

Comment:
Doesn't this require the registry?

Reply:
@ChristopherDavenport The apply method requires the registry, the `build` methods creates a new registry.

Reply:
In which case this would need to be first and be the registry used for metering.

Reply:
I totally missed the difference between build and apply. I can fix that up.